demisto_sdk/commands/test_content/mock_server.py

```diff
@@ -251,6 +251,9 @@ def push_mock_files(self):
                 f"Failed pushing mock files with error: {exc}", real_time=True
             )
         finally:
+            self.logging_module.debug(
+                "Successfully pushed new/updated mock files to mock git repo.", real_time=True
```

Comment:
Do we actually know it was successful if it is in the finally section?

Reply:
No, we don't. While there are logs for when an Exception is caught, we can't reliably confirm the success of the push, which might be misleading. Removing this log.